packages/lisk-transactions/test/utils/validation/validator.js

```diff
@@ -0,0 +1,1019 @@
+			expect(errors[0].dataPath).to.equal('.signature');
+			return expect(valid).to.be.false;
+		});
+
+		it('should validate to be false with errors when signSignature is not valid signature', () => {
+			const invalidTransaction = Object.assign({}, validTransaction, {
+				signSignature: 'signature',
+			});
+			const { valid, errors } = validateTransaction(invalidTransaction);
+			expect(errors[0].dataPath).to.equal('.signSignature');
+			return expect(valid).to.be.false;
+		});
+
+		it('should validate to be false with errors when signatures is not an array', () => {
+			const invalidTransaction = Object.assign({}, validTransaction, {
+				signatures: 'signature',
+			});
+			const { valid, errors } = validateTransaction(invalidTransaction);
+			expect(errors[0].dataPath).to.equal('.signatures');
+			return expect(valid).to.be.false;
+		});
+
+		it('should validate to be false with errors when signatures contain duplicate elements', () => {
+			const invalidTransaction = Object.assign({}, validTransaction, {
+				signatures: [
+					'fad7b3b31c337b39190d206831c1eaadc6bbf3878a3507a868a5fbb03471b383042bf3bb7cee20d9844f2f4d1bb90d08bc3589b8b7d27a538be285deec7a9504',
+					'fad7b3b31c337b39190d206831c1eaadc6bbf3878a3507a868a5fbb03471b383042bf3bb7cee20d9844f2f4d1bb90d08bc3589b8b7d27a538be285deec7a9504',
+				],
+			});
+			const { valid, errors } = validateTransaction(invalidTransaction);
+			expect(errors[0].dataPath).to.equal('.signatures');
+			return expect(valid).to.be.false;
+		});
+
+		it('should validate to be false with errors when signatures contain an invalid signature', () => {
+			const invalidTransaction = Object.assign({}, validTransaction, {
+				signatures: [
+					'fad7b3b31c337b39190d206831c1eaadc6bbf3878a3507a868a5fbb03471b383042bf3bb7cee20d9844f2f4d1bb90d08bc3589b8b7d27a538be285deec7a9504',
+					'zzzzzzzzzzzzzzzzzzzzzz6831c1eaadc6bbf3878a3507a868a5fbb03471b383042bf3bb7cee20d9844f2f4d1bb90d08bc3589b8b7d27a538be285deec7a9504',
+				],
+			});
+			const { valid, errors } = validateTransaction(invalidTransaction);
+			expect(errors[0].dataPath).to.equal('.signatures[1]');
+			return expect(valid).to.be.false;
+		});
+
+		it('should validate to be true when signatures is contains valid signatures', () => {
+			const invalidTransaction = Object.assign({}, validTransaction, {
+				signatures: [
+					'fad7b3b31c337b39190d206831c1eaadc6bbf3878a3507a868a5fbb03471b383042bf3bb7cee20d9844f2f4d1bb90d08bc3589b8b7d27a538be285deec7a9504',
+					'aaaab3b31c337b39190d206831c1eaadc6bbf3878a3507a868a5fbb03471b383042bf3bb7cee20d9844f2f4d1bb90d08bc3589b8b7d27a538be285deec7a9504',
+				],
+			});
+			const { valid, errors } = validateTransaction(invalidTransaction);
+			expect(errors).to.be.null;
+			return expect(valid).to.be.true;
+		});
+
+		it('should validate to be true when data is 64 bytes', () => {
+			const invalidTransaction = Object.assign({}, validTransaction, {
+				asset: {
+					data: 'fad7b3b31c337b39190d206831c1eaad',
```

Comment:
This should have a length of 64 and I think it's confusing if it only has hex characters.